shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -329,6 +335,15 @@ - (void)dealloc {
   _engine.viewController = nil;
 }
 
+- (void)setupNotificationCenterObservers {
```

Comment:
If you keep this (as a function), note that "setup" is a noun, while the verb is "set up", so methods/functions should be capitalized accordingly. Also, this needs a declaration comment.